Resources/public/paella/player/config/profiles/profiles.json

```diff
@@ -8,50 +8,12 @@
 		"masterVideo":{"content":"presentation","rect":[
 			{"aspectRatio":"16/9","left":"450","top":"135","width":"816","height":"459"}],"visible":"true","layer":"1"},
 		"blackBoardImages":{"left":"10","top":"325","width":"432","height":"324"},
-		"background":{"content":"slide_professor_paella.jpg","zIndex":5,"rect":{"left":"0","top":"0","width":"1280","height":"720"},"visible":"true","layer":"0"},
```

Comment:
Idea: create new profiles dir like `pr` and don't change original paella profiles